from moncash.exceptions.unexpected_error import UnexpectedError 

class ConnectionError(UnexpectedError):
    pass